optics/src/Optics.hs

```diff
@@ -205,10 +208,34 @@ module Optics
   --
   , module Optics.Indexed
 
-  -- * At
+  -- * Optics utilities
+
+  -- ** At
+
+  -- | An 'AffineTraversal' to traverse a key in a map or an element of a
+  -- sequence:
+  --
+  -- >>> headOf (ix 1) ['a','b','c']
```

Comment:
Using `preview` would be more clear here :)

Reply:
Thanks, good point. That's what I get for trying to think of examples late at night.